website/docs/reference/resource-configs/invalidate_hard_deletes.md

```diff
@@ -1,14 +1,22 @@
 ---
-title: invalidate_hard_deletes (legacy)
+title: invalidate_hard_deletes
 resource_types: [snapshots]
 description: "Invalidate_hard_deletes - Read this in-depth guide to learn about configurations in dbt."
 datatype: column_name
-sidebar_label: invalidate_hard_deletes (legacy)
+sidebar_label: invalidate_hard_deletes
 ---
 
+# invalidate_hard_deletes <Lifecycle status="legacy" />
+
+<IntroText>
+
+Legacy opt-in configuration to enable invalidating hard deleted records while snapshotting the query.
+
+</IntroText>
+
 :::warning This is a legacy config &mdash; Use the [`hard_deletes`](/reference/resource-configs/hard-deletes) config instead.
 
-In Versionless and dbt Core 1.9 and higher, the [`hard_deletes`](/reference/resource-configs/hard-deletes) config replaces the `invalidate_hard_deletes` config for better control over how to handle deleted rows from the source. 
+In dbt Cloud release tracks and dbt Core 1.9 and higher, the [`hard_deletes`](/reference/resource-configs/hard-deletes) config replaces the `invalidate_hard_deletes` config for better control over how to handle deleted rows from the source. 
 
 For new snapshots, set the config to `hard_deletes='invalidate'` instead of `invalidate_hard_deletes=true`. For existing snapshots, [arrange an update](/reference/snapshot-configs#snapshot-configuration-migration) of pre-existing tables before enabling this setting. Refer to 
```

Comment:
```suggestion
For new snapshots, set the config to `hard_deletes='invalidate'` instead of `invalidate_hard_deletes=true`. For existing snapshots, [arrange an update](/reference/snapshot-configs#snapshot-configuration-migration) of pre-existing tables before enabling this setting. 
```